examples/example.oas.yaml

```diff
@@ -31,6 +31,7 @@ components:
         published:
```

Comment:
Please revert all the changes in this file. This should be an exact copy of "aep/general/example.oas.yaml" in the aep.dev repo. When a rule detects issues in this file (e.g. path not being required), we need to a) confirm that the issue is a true positive, and if so b) update the aep tooling to correct the problem and then c) copy the updated example.oas.yaml here.